lib/Jsrt/Jsrt.cpp

```diff
@@ -5329,4 +5329,39 @@ CHAKRA_API JsSetHostPromiseRejectionTracker(_In_ JsHostPromiseRejectionTrackerCa
     /*allowInObjectBeforeCollectCallback*/true);
 }
 
+CHAKRA_API JsGetProxyProperties (_In_ JsValueRef object, _Out_ bool* isProxy, _Out_opt_ JsValueRef* target, _Out_opt_ JsValueRef* handler)
+{
+    return ContextAPINoScriptWrapper_NoRecord([&](Js::ScriptContext * scriptContext) -> JsErrorCode {
+        VALIDATE_INCOMING_REFERENCE(object, scriptContext);
+        PARAM_NOT_NULL(isProxy);
+
+        *isProxy = false;
```

Comment:
This initial assignment is probably unnecessary.

Reply:
I was following the pseudo-policy I've seen of always initialising out values. But in this case considering it's set to a bool on the next line I can see your point - changed.

Reply:
@rhuanjl The idea I think is that out parameters are always written to regardless of which codepath you take (in languages like C# with native support for out params it’s actually an error not to do so).  If, like here, the value is unconditionally set to something then initializing it is redundant.

Reply:
I think SAL/Prefast is supposed to give errors for things like that, but I am not sure if it catches everything.